nomad/leader.go

```diff
@@ -318,6 +323,60 @@ func (s *Server) establishLeadership(stopCh chan struct{}) error {
 	return nil
 }
 
+func (s *Server) initializeMasterToken() error {
+	if !s.config.ACLEnabled {
+		return nil
+	}
+
+	if s.config.Region != s.config.AuthoritativeRegion {
+		return nil
+	}
+	if master := s.config.ACLMasterToken; len(master) > 0 {
+		state, err := s.State().Snapshot()
+		if err != nil {
+			return err
+		}
+		token, err := state.ACLTokenBySecretID(nil, master)
+		if err != nil {
+			return fmt.Errorf("failed to get master token: %v", err)
+		}
+		if token == nil {
+			token = &structs.ACLToken{
+				AccessorID: uuid.Generate(),
```

Comment:
Would it be useful to allow the AccessorID to be specified in the HCL as well? Consul doesn't for their `master_token`, but they do allow it for subsequent tokens created via their API in order to ease automation.